packages/core/src/RenderingEngine/helpers/setDefaultVolumeVOI.ts

```diff
@@ -166,7 +182,6 @@ async function getVOIFromMinMax(
     },
     priority: PRIORITY,
     requestType: REQUEST_TYPE,
```

Comment:
This was correct. We should include `useNativeDataType`. I tried your data without changing this part, and it worked fine. Can you revert this change?

Reply:
As parameter useNativeDataType is not described in JSDoc, It updates the comment only. The codes is not changed. Do I need revert it?

Reply:
I believe you are referring to a different location, as this is the code modification.

![CleanShot 2024-05-17 at 08 57 42@2x](https://github.com/cornerstonejs/cornerstone3D/assets/7490180/c945284a-a48c-4956-8d0b-1c0b43ce6808)

Reply:
It's my mistake. Could you review again